astroquery/linelists/jplspec/core.py

```diff
@@ -4,8 +4,8 @@
 
 import astropy.units as u
 from astropy.io import ascii
-from ..query import BaseQuery
-from ..utils import async_to_sync
+from ...query import BaseQuery
+from ...utils import async_to_sync
```

Comment:
as you already touch this, use normal imports, we don't need absolute ones, especially not for anything we don't directly test in this particular file